lib/assignDisabledRanges.js

```diff
@@ -2,12 +2,12 @@
 "use strict"
 const _ = require("lodash")
 
-const COMMAND_PREFIX = "stylelint-"
-const disableCommand = COMMAND_PREFIX + "disable"
-const enableCommand = COMMAND_PREFIX + "enable"
-const disableLineCommand = COMMAND_PREFIX + "disable-line"
-const disableNextLineCommand = COMMAND_PREFIX + "disable-next-line"
-const ALL_RULES = "all"
+const COMMAND_PREFIX: string = "stylelint-"
```

Comment:
@ivanzusko The problem here is that our Flow annotations need to use [the comment syntax](https://flowtype.org/docs/syntax.html#comment-syntax), to avoid the need to compile the code when running it.